### DISCUSSION TITLE
New monsters

### DISCUSSION THREAD

**Discussion by HippasusTwo** [Tuxemon/Tuxemon#2468] [Ideas]
This is a thread for making entirely new monsters.

Currently, there is focus on creating evolutions for the baby-looking 'student starter trio' (like Pharfan) and Terrex, and some discussion of making new evos for Doctsky - this is a continuation of some discussion on the 'Monsters Near Completion' thread. No art (yet) exists for these suggested evolutions, although I will probably make some for Doctsky. Some names have been suggested for most of them but Terrex's hypothetical evo (or evos) does not have a name yet.

If you are uninterested in that set, it might be nice to have some monsters that cover concepts that have not been done by other monsters yet. For instance, a dimetrodon (or another Permian era creature, though we do have one dicynodont), 'fish evolving to walk on land' / Devonian inspired, tree frogs, or some less famous marsupials like Tasmanian tigers and wallabies (we have koalas and platypus). There is also a surprising lack of ghosts that I have noticed besides one 'samurai ghost', though I suppose possessed inanimate objects count. Very few microbes have been done so there's a lot of room for inspiration based on weird microbes.

Edit to copy some details from the other thread:
Jaskrendix suggested some names:
Hampterra, for Hampotamos, may be a terrain based evo, might be more humanoid. I suggested possibly an Egyptian Goddess inspiration. Possibly Demostylian, relative of hippos.
Pharavion, with Pharfan acting like 'Magikarp' and being weak until it evolves by level. Possibly one of the weirder extinct elephants like Deinotherium, flying with wings or magic carpet.
'Rinocereed's evolution could be a more armored, adult form with a distinctive horn or crest. "Rinokron" could be its evolved form, with a moveset that emphasizes defense and strength. Its unique morph process might involve Rinocereed being trained in a specific type of technique (not in the party, but for the specific monster).' Possibly based on some fun rhino relatives like Paraceratherium, although others had more interesting horns and crests. Some even had multiple!

I suggested for the hyena evos:
Askylpaws, a reference to Asclepius the Greek Mythological Doctor that could heal the dead back to life while playing on the previous evolution's name.
Erythroskyte, it could have spots shaped like red blood cells. It might be good to do evolution by gender - male hyenas are smaller and less aggressive than female hyenas. Then again, most people still can't tell them apart, so perhaps an item evo would be better.

**Comment by Davidliang11292** [comment]
I also have some inspirations for new Tuxemon, but I'm sorry that I can't draw.

My inspirations come from toys:

The first Tuxemon I designed is inspired by building blocks. Its original form is a single block; after its first evolution, it becomes some assembled blocks; and its final evolved form is a colorful, cute Tuxemon resembling a horse.

The second Tuxemon I designed draws inspiration from jigsaw puzzles. Its original form is a single puzzle piece; after its first evolution, it becomes a partially completed puzzle; and its final evolved form is a complete puzzle. There are many types of these puzzle Tuxemon, each with different colors and attributes (for example, an orange puzzle is a fire-type Tuxemon, while a blue puzzle is a water-type Tuxemon), but their stats are generally low.

The third Tuxemon I designed is inspired by teddy bears. Its original form is a damaged teddy bear; after its first evolution, it becomes a patched-up teddy bear; and its final evolved form is a teddy bear wearing clothes.

**Reply by HippasusTwo** [reply]
Jigsaw like this?
![puzzled](https://github.com/user-attachments/assets/f643d90e-1b83-43da-bb96-b6a8866f1f05)
![puzzlekey](https://github.com/user-attachments/assets/d4c306ca-bcb6-4108-9822-cf7ecb2a7a29)
![puzzleking](https://github.com/user-attachments/assets/6e5909df-5949-4b48-bf5b-098f09303369)
![puzzlewhite](https://github.com/user-attachments/assets/acf00bac-5ae2-4e5e-ad55-a3e318bbd3e2)

**Reply by Davidliang11292** [reply]
Yes, just like that! thank you

**Reply by HippasusTwo** [reply]
Alrighty. To be actually added to the game, they need names, a blurb like 'Lost jigsaw pieces can become inhabited by curious ghosts and become MONSTER NAME.' or something, and evolution method, like level or having another of its species in the same party. And also back and face sprites but those wouldn't take very long for such simple sprites. I assume their moveset when they evolve to have lots of puzzle pieces would have moves of all elements and be very diverse, making up for their low stats.

Maybe Puzzleuno (Uno means one in Spanish) or Puzzled, Puzzlekey or Puzzleduo, and Puzzleking? 

Finally, type. The simplest way to implement different types at the moment would be making each color base sprite a different variety, but then you'd end up having them evolve to the same 'mon which I'm not sure if is allowed. Alternatively if color variants get implemented, it could be cool to have variants optionally tie to different types. For now, it might be easiest just to pick one like White (steel?) for the first form.

---
About your other mon: a patched up Teddy Bear makes me think of animals that have different colored patches, like a panda bear, sun bear or Spectacled Andean bear. Pandas originally evolved from normal looking bears, so that could be an interesting nod to history.

**Comment by HippasusTwo** [comment]
![askylpaws_and_erythroskyte](https://github.com/user-attachments/assets/b527efeb-e18c-4971-a041-e03aaa2dcb91)
Some concept art for Askylpaws and Erythroskyte/Erythrosky of varying quality.
Let me know if you guys have any preferences for one concept over another. As you can see I've started on some spritework for Erythrosky, one is a bit more 'realistic' and might be suitable for a quadrupedal Askylpaws, but I am debating making a biped holding a staff or something, or maybe a giant syringe. And possibly with snakes coming out of the body, head would be a nice Medusa ref. Could go syringe tail for Erythrosky and snake tail for Askylpaws
Alternate colors are possible, but I'd rather like to have Erythrosky some shade of pink close to its origin form. A black deathly form might be interesting for Askylpaws, I just chose purple here because it's not too far off from pink.

**Reply by JaskRendix** [reply]
It looks good the purple version, it gives a distinct identity and sets it apart from the other color scheme. If you decide to go with the bipedal version, I think a simple staff or log could work well, especially if the syringe design is proving tricky. The staff or log could be a nod to the traditional view, and it would be interesting. Medusa head too!

**Reply by HippasusTwo** [reply]
Alright, front sprite for Erythrosky(te) is basically complete, I decided to give it the syringe, so Askylpaws will have snake(s) and staff. Medusa head might be hard to fit in a tiny 64 by 64 sprite, we'll see. Although there is also a normal tailed non-syringe version:
![erythroskyte-normaltail](https://github.com/user-attachments/assets/e72867dc-5e88-4cb6-953e-b13d4cb8304b)
![erythroskyte-front](https://github.com/user-attachments/assets/e740b76d-d161-40c8-bd44-3416bb2803cc)
Playing with orientations of the staff and body for Askylpaws:
![askylpaws](https://github.com/user-attachments/assets/3f9a65f6-2f8f-43c0-a15f-d9d17fce9890)

I think if I want the medusa head to work I might need to redraw the head a bit / possibly re-orient it more toward the middle for more room to draw snakes, although it would still have little room on top (it's already a tall sprite). And maybe for bipedal reshape it a bit less 'realistic' / more humanoid, but I guess it depends on how much effort I want to spend on it.
The quadruped version with the staff in mouth looks fairly nice, I just worry it might be a little too similar to Erythroskyte.

**Reply by Sanglorian** [reply]
Erythroskyte is looking good! 

Personally I like the staff in mouth, no Medusa-head, standing on hind legs version for Askylpaws. But I think any configuration you come up with will look good.

**Reply by HippasusTwo** [reply]
![askylpaws-front](https://github.com/user-attachments/assets/c6a130ab-6a8e-406c-bbb7-531d699c0ee2)
![askylpaws-front-medusa](https://github.com/user-attachments/assets/81cf77f0-e059-4909-a7de-42383f7521b2)
I decided to try both versions in that pose. Medusa head may be a bit crowded: it's a very full image. So I'm leaning that way also.

**Reply by Sanglorian** [reply]
Lovely! Agree about the crowdedness.

**Reply by HippasusTwo** [reply]
![askylpaws-back](https://github.com/user-attachments/assets/e60fac62-6145-4453-abb3-0e117d4dd822)
![erythroskyte-back](https://github.com/user-attachments/assets/bbff1d81-c089-4fe1-8e5e-e1de6ce5fad8)
![erythroskyte-back2](https://github.com/user-attachments/assets/fe10eb92-0940-4c6a-b8bc-d6b8a665b755)

**Comment by HippasusTwo** [comment]
Since amphibians are so unloved, I did some doodles for some possible concepts. Feel free to add your own.

Neonewt, play on newt and neonate, and 'neo' means new which is contained in Newt. It's a mixture of tadpole and newt larvae.
Variant name: Neonew
could evolve into a bunch of different things. Probably Wood/Water or Earth/Water.

Earthy-woody looking frog and toad evos. Earth/water in case someone later does a proper tree frog (which I imagine would be wood/water). Maybe have ability to make magical screens that they can climb on if they can't find trees. 

fire sala 'mender' salamander that likes to mend, and a maybe-winged evo, maybe steel/fire (because needles). Could have a needle technique and share it with Doctsky and cactuses (Hoodoll)?

'Bomber' dart frog, Poisonous and explosive, maybe water/steel?

![amphibians](https://github.com/user-attachments/assets/fcddaf10-05bb-4977-b95e-e8ba76ab9fa9)

**Reply by Sanglorian** [reply]
I really enjoy these, thanks for putting them together!

**Reply by JaskRendix** [reply]
Some attempts:
<img width="64" height="64" alt="bombatoad2" src="https://github.com/user-attachments/assets/b4158cf6-2758-4137-b545-cdb615ea29af" />
<img width="64" height="64" alt="bombatoad1" src="https://github.com/user-attachments/assets/4fe2cafa-133e-42b3-a53a-9b3a23e359ad" />
<img width="64" height="64" alt="salamander" src="https://github.com/user-attachments/assets/c2145bf4-e406-4f08-9936-fc50601368b3" />

**Reply by HippasusTwo** [reply]
That's really cool! I did imagine, however, the little red guy as having only two limbs, because he/she is a tadpole transforming into a frog, and they actually don't get all their limbs at once if I recall correctly (I haven't raised tadpoles since I was a small child).
edit: however, it really doesn't matter too much. One could even simply make the 'tadpole form' a starter stage and the four limbed all red guy a middle stage, and have it independent from neonewt. The only reason I had a tadpole evolving from a newt larvae is that this reflects evolutionary history where newt like forms came before frog tadpoles did.

I see you added Neonewt and Skymander, but none of the others, correct? I wouldn't want to accidentally re-add the same monster.

**Comment by JaskRendix** [comment]
from [here](https://github.com/Tuxemon/Tuxemon/discussions/2337#discussioncomment-11000532): a 'Eukaryote' primitive single/multicellular could evolve into basically anything. It could be named **Primordia**:
- blurb: a primitive, single-celled organism that is said to be the building block of all life. It has a gelatinous, amoeba-like body that can change shape at will, allowing it to adapt to its surroundings.
- details: despite its simplicity, Primordia possesses a unique genetic makeup that allows it to evolve into a wide variety of forms, making it a highly sought-after specimen among scientists and trainers. It is able to learn and remember its surroundings, and can even communicate with its trainers through a series of chemical signals. Primordia can be found in a wide variety of environments, from the depths of the ocean to the surface of the land. It is able to survive in extreme conditions, from freezing cold to boiling hot, and can even thrive in environments with high levels of radiation. It is able to absorb nutrients from its surroundings, and can even photosynthesize (I'm still unsure about it) to some extent. 
- techniques: like "Splash" + regeneration of some sort, so 2 techniques
- appearance: it appears as a small, translucent blob with a shimmering, iridescent sheen. It has no discernible eyes.
- height: approximately 0.5 meters
- weight: approximately 0.2 kilograms

**Reply by HippasusTwo** [reply]
If we're going with 'building block of all life', then it's actually even more primitive than an eukaryote. You're talking about a LUCA (Last Universal Common Ancestor) candidate, the ancestor of both Bacteria, Archaea and Eukaryotes (We're Eukaryotes, which may have come about from a fusion between Bacteria and an Archaea, giving us mitochondria that help us breathe oxygen; another fusion with Photosynthetic Bacteria and an Eukaryote gave photosynthesis, but you don't need to be totally realistic if you don't want). LUCA is indeed believed to have been able to survive extreme environments like hot hydrothermal vents.

An interesting characteristic of early life forms is 'horizontal gene transfer'. In other words, they can steal techniques! So I think it should have a copycat/sketch sort of technique or techniques. Another characteristic is that they spontaneously split into two, duplicating themselves. That could make for a good move.

They did *not* like oxygen / air. So probably it should be at least part water type. Water/Fire maybe?

**Reply by JaskRendix** [reply]
Primordia being even more primitive than an eukaryote. That's a great point, and it could add an extra layer of depth to its concept.

I like the idea of incorporating horizontal gene transfer into its techniques. A "Copycat" or "Mimic" move could be a great way to reflect this ability. Perhaps it could even have a chance to learn a random technique from its opponents, making it a bit unpredictable.

The idea of spontaneous duplication is also fascinating. A move like "Split" or "Duplicate" could be a great way to represent this ability. It could even have a chance to create multiple copies of itself, making it harder to hit.

Water/Fire is an interesting combination.

Ideas:
* Appearance: same as before, but with a slightly more primitive or amoeba-like appearance to reflect its status as a LUCA candidate.

Techniques: https://wiki.tuxemon.org/User:Jaskrendix#ideas_techniques_+_mechanics

**Reply by HippasusTwo** [reply]
![microbe](https://github.com/user-attachments/assets/42362435-9ac7-469c-8bc0-966580816522)

**Reply by JaskRendix** [reply]
It looks really good!

**Reply by HippasusTwo** [reply]
Alright, sprite set for it is done. Added to wiki. It needs moves and a height.

**Comment by HippasusTwo** [comment]
Pharavion maybe. A bit unpolished.
![pharavion-front-singletusk](https://github.com/user-attachments/assets/9e9afb1d-ff48-484d-a880-f11b9799585f)
![pharfanevo-front-multitusk](https://github.com/user-attachments/assets/7593571b-46f9-4772-9677-d24bb81b9cf4)

edit:
I decided to go with bluer color, touched up the carpet to look nicer. Finished the set.
![pharavion-back](https://github.com/user-attachments/assets/a13ba083-617e-4a5d-a2df-ac12e412f61f)

**Comment by HippasusTwo** [comment]
This is a random creature set that popped in my head that I quick-doodled. It doesn't really fill a need per say, except for the fact that rodents are extremely common animals so you would expect a fair number of them. An alternative name for the middle evo was 'Harvester', playing on harvest mice and harvests being associated with using scythes. The original version I had in my head had a skull motif and bone markings, but I decided to drop that since we have a gopher-mouse(?) with that now. I played with making both long eared and short eared versions of Scytherat, with the long ears meant to vaguely evoke devils but mostly ended up reminding me of foxes.
![scytheratfam](https://github.com/user-attachments/assets/87afbde5-5e79-4cfe-bbc6-dbae13114ebd)
![scytherat3](https://github.com/user-attachments/assets/830222d1-82b3-4f28-b4aa-c88cae35c6db)

**Reply by JaskRendix** [reply]
The decision to avoid the skull/bone motif is a good one. It allows your creatures to have a distinct identity. How do you envision the long-eared variant looking and behaving? Are the long ears primarily an aesthetic choice, or do they serve a functional purpose in-game?

As backstory/ lore could be considered pests by farmers, but also play an important role in the ecosystem by aerating the soil.

What do you think about variants? A desert one could be sandy-colored, withstand heat, and maybe even cause sandstorms. An arctic one could have thick fur, use ice attacks, and survive the cold.

These creatures could burrow and maybe have a poisonous bite.

**Reply by HippasusTwo** [reply]
Could be both: I imagined that like lab/dumbo rats, sometimes variants get bred for their looks, but in nature, you see longer ears associated with things like living in a desert. So I could make a desert one sandy colored and long eared. While an icy one would have vole-like ears covered in fur so they're harder to see.
![scytheratvariants](https://github.com/user-attachments/assets/54c8c864-7e61-4a0a-9909-40c25e1c327f)

**Comment by HippasusTwo** [comment]
Ferricran alt form or possible evo with wings? Was inspired by the talk about it having wings once in its page. Not too many 'four evolution evos', but if anything deserves a secret fourth evo, I guess it'd be a dragon.
![mythricran-front](https://github.com/user-attachments/assets/c31f3c0e-e48d-4aa0-b6ca-56d21aacfc0a)

**Reply by Sanglorian** [reply]
Nice one! There's an idea from years ago on the Tuxemon forums that we never really developed, that some monsters would morph for a single combat. (We were thinking of it for the Cathedral starters - the idea being that the corporations liked the idea of a single-time morph because people would keep having to buy the morph items). 

That could be a good way to give a three-morph family a "fourth-morph" - a temporary, in-combat morph only. It could be triggered by an item, or by a particular technique, or by a particular condition being met (e.g. Ferricran uses a melee technique on a Metal-element mon).

**Reply by JaskRendix** [reply]
Just to clarify, are you thinking about the monster actually evolving during the combat, or do you mean using the item first, then the monster fights in its new form, and afterwards it goes back to its original form (or something else)? I'm curious about how much time and effort it might waste to implement this (depending on the scenario)

**Reply by Sanglorian** [reply]
> Just to clarify, are you thinking about the monster actually evolving during the combat, or do you mean using the item first, then the monster fights in its new form, and afterwards it goes back to its original form (or something else)? I'm curious about how much time and effort it might waste to implement this (depending on the scenario)

What I had in mind was that the evolution would take place in combat, so for example for an item triggering a temporary morph it would work something like this: 

1. Combat begins. 
2. The player can use items, like normal (potions, capture devices, etc). If they choose a particular Temporary Morph item and apply it to an eligible 'mon, the 'mon evolves right away (maybe the evolution animation shows, maybe it just changes to save time and make things simpler)
4. Maybe evolving makes an additional technique available? For example Winged Ferricran may get Altitude.
5. At the end of the combat, the evolution ends (maybe there's a snippet of text: "Winged Ferricran returns to its previous form." or something like that). The monster returns to its pre-combat move list, stats, etc (but they keep XP for the encounter).  

But that's just what I had in mind - I'm flexible if there's other ways of doing it. 

And while I think it'd be a fun addition, please don't feel obliged to implement it! Just an idea I've had kicking around.

**Reply by HippasusTwo** [reply]
For a change in techniques and stats, it sounds like it would be easiest to load the extra form as a different species and grab its move table, but, that could be disruptive if the user had a custom moveset.
What one could do is have a move that changes depending on the user's type, or simply always matches the user's type (so it always hits with STAB / same type attack bonus). 'Inner Element' could be a good name for it, and then there would be a counterpart ranged version, 'Elemental Breathe' or the like, and perhaps a more powerful version for late game. I think this would be useful even without form changes, because it'd make a great TM that is guaranteed to be learnable by one/all of your monsters and gives type coverage. Your monster accidentally not know any of its types moves or only ranged/melee of that type? No prob, just use the TM!

**Comment by HippasusTwo** [comment]
![hampterra](https://github.com/user-attachments/assets/45bf68e8-089e-464d-84ea-6ff01611fb80)

Quickie possible Hampterra doodle concepts. The protruding out teeth are a nod to demostylians.

**Reply by HippasusTwo** [reply]
<img width="64" height="64" alt="hamp" src="https://github.com/user-attachments/assets/72a975d5-c292-4763-b487-7da2f6ac56ec" /> try 1, don't really like it... altho' it works if you want a goofy light hearted evo to match original.
<img width="64" height="64" alt="hamp" src="https://github.com/user-attachments/assets/2ce515bd-2668-41a6-b2cd-6fdafc27a7e6" />
try 2, more serious.
ah, just realized I have it facing the wrong way... try 3
<img width="64" height="64" alt="hampterra-front" src="https://github.com/user-attachments/assets/0633af25-268a-492e-a9d4-fca3d566d717" />
I think I really like that one.

**Comment by JaskRendix** [comment]
[Psilopignon](https://wiki.tuxemon.org/Psilopignon)

**Reply by HippasusTwo** [reply]
We could use more mushrooms. I also like the idea of having another psychic. The Pollen Plast animation could be re-used for a Fungal Spore move that fits it thematically, or several. Hallucigenic Spore?
There's an explosion animation or two that has never been used I think, it could be Mind Explosion, a 3.0 power move.

It could also have a 'Poisonous' tag, meaning you take damage if you use a 'Bite' tagged move on it?

https://en.wikipedia.org/wiki/Psilocybin_mushroom#/media/File:Psilocybe_Mushrooms_statues.jpg
Mushroom stones inspired by them have an interesting appearance. I like the strange diamond like pattern etched on the one furthest to the right.

**Reply by JaskRendix** [reply]
I'm glad you see the potential for more mushroom-themed creatures and psychic types. "Fungal Spore" or "Hallucigenic Spore" would fit perfectly, and it's a great way to utilize existing assets creatively.

I don't think a 'Poisonous' tag fits well with the concept of a psychic mushroom. Instead, how about considering tags like 'Hallucinogenic' or 'Mind-Altering'? These could reflect its psychic abilities and the idea of inducing effects on the opponent without directly causing damage. Additionally, we could introduce a 'Spore' tag that allows it to inflict status conditions like confusion or sleep, which aligns more with its theme. What do you think?

The mushroom stones are a great visual reference. I agree about the diamond-like pattern on the one furthest to the right. It has a really interesting and unique appearance. I would love to see that pattern worked into the monster's design, perhaps as a texture on its cap or as glowing markings.

Could Psilopignon be connected to other psychic or plant-based creatures?
Are there any other creatures that are effected by the spores (or immune - tag: "spore")?
Regarding "Mind Explosion" should it always inflict self-damage, or could there be a chance to avoid it based on Psilopignon's stats or a specific condition or tags?
Regarding "Hallucigenic Spore" what kind of status effects do you envision it inflicting? Should it be a random chance of different effects (confusion, paralysis, sleep)?

**Reply by HippasusTwo** [reply]
The existing 'Mental' (or Psychic if it gets renamed) would fit with 'mind altering' or psychic abilities just fine, and is an existing tag. Some kind of status tag to supplement that would be good, and 'Spore' could work as that label.
edit because I'm sleep deprived: Hallucigenic to induce some sort of mind altered status on bite could be an interesting tag. Confusion?

Some bacteria produce spores. Most plants don't but non-seed plants like ferns might. Lichens are a hybrid organism that are part fungi, I think I spotted one character that reminded me of a Lichen, the rock plant little dude Tarpeur. So we might give them both the Spore tag. I don't think we have any ferns off the top of my head.

I'm happy either way, I created the move Mind Explosion so feel free to edit it.
I think random chance could be fun. Do we have a 'Random' effect yet? I'll just put it down as confusion for now.

**Reply by HippasusTwo** [reply]
@JaskRendix 
The similar name makes me think it's supposed to be an evo of Cowpignon, was that intended or was that just a coincidence of same root language?
It looks like it's very close to finished, does it just need a move-set and a number?

**Reply by JaskRendix** [reply]
@HippasusTwo yep, Psilopignon's ready to roll! I did think about making it an evo of Cowpignon at first, felt like a natural fit with the name and vibe, but in the end I figured it worked better as a separate fungus entirely. So yeah, the similar name’s just a coincidence from the same root language. I might still hint at some kind of relationship between them, like a lore connection, but they’re standing on their own as distinct Tuxemon.

Also, thanks for the moves you added on Tuxepedia! I was kinda holding off on finalizing a few things while waiting for @Sanglorian to check the newer entries, there’s been a bunch lately, like the new Vivipere evos and others popping up (Emulon, Torrentaur, Gnu, etc). Looking forward to seeing how it all settles!

**Reply by HippasusTwo** [reply]
OK, thanks for telling me, in that case, the movesets are too similar and one or the other of them needs to get slightly different moves. I think perhaps Mind Vise would be a natural choice.

**Comment by HippasusTwo** [comment]
<img width="64" height="64" alt="rinokron" src="https://github.com/user-attachments/assets/4708f56f-a21c-481e-9152-c10f82e16393" />
I forgot I was supposed to make Rinokron (rinoceered evo) armored. Oh well.

It seems like the two types with the lowest membership counts are electric and frost, if anyone wants to make one of those. In particular, a lot of the electric are part water, and a lot of the frost are part wood, so it would be good to have more variety that isn't one of those duals.
